### DISCUSSION TITLE
[Script request] slskd

### DISCUSSION THREAD

**Discussion by joon-im** [community-scripts/ProxmoxVE#735] [Request script]
### Application Name

slskd

### Website

https://github.com/slskd/slskd

### Description

A modern client-server application for the [Soulseek](https://www.slsknet.org/news/) file-sharing network. I find it is pretty useful for finding music. Thanks! 

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by Daviid-P** [comment]
I recommend docker, make a stack of slskd + soularr

**Reply by Jokosch** [reply]
Hello David,

My current Problem is that I have a BTRFS Drive.
I can Access the BTRFS Drive without any Problem from any LXC Container after I configured the Mount Point in the Config File of the Container.
Docker doesn't support BTRFS out of the box and I tried this Guide " https://docs.docker.com/engine/storage/drivers/btrfs-driver/ " several Times and it doesn't work.

That is why I would still like to see a slskd LXC Script in the future if its possible.

Greetings 

Jokosch

**Comment by lklynet** [comment]
I just came here looking for the same thing. I'd love an slskd script, especially if it can be bundled with soularr!

**Reply by IReclaimer** [reply]
+1 for this idea, it would be excellent to have them bundled together

**Reply by vhsdream** [reply]
I have an LXC with Lidarr, slskd and Soularr. I'll put this on my to do list in the next few weeks, or perhaps someone else will have something ready before then 😄

**Comment by vhsdream** [comment]
Finally merged today 👍🏻  #3516
